src/libs/ReportUtils.js

```diff
@@ -917,36 +917,34 @@ function buildOptimisticChatReport(
  */
 function buildOptimisticCreatedReportAction(ownerEmail) {
     return {
-        0: {
-            actionName: CONST.REPORT.ACTIONS.TYPE.CREATED,
-            reportActionID: NumberUtils.rand64(),
-            pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
-            actorAccountID: currentUserAccountID,
-            message: [
-                {
-                    type: CONST.REPORT.MESSAGE.TYPE.TEXT,
-                    style: 'strong',
-                    text: ownerEmail === currentUserEmail ? 'You' : ownerEmail,
-                },
-                {
-                    type: CONST.REPORT.MESSAGE.TYPE.TEXT,
-                    style: 'normal',
-                    text: ' created this report',
-                },
-            ],
-            person: [
-                {
-                    type: CONST.REPORT.MESSAGE.TYPE.TEXT,
-                    style: 'strong',
-                    text: lodashGet(allPersonalDetails, [currentUserEmail, 'displayName'], currentUserEmail),
-                },
-            ],
-            automatic: false,
-            sequenceNumber: 0,
-            avatar: lodashGet(allPersonalDetails, [currentUserEmail, 'avatar'], getDefaultAvatar(currentUserEmail)),
-            created: DateUtils.getDBTime(),
-            shouldShow: true,
-        },
+        reportActionID: NumberUtils.rand64(),
+        actionName: CONST.REPORT.ACTIONS.TYPE.CREATED,
+        pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
+        actorAccountID: currentUserAccountID,
+        message: [
+            {
+                type: CONST.REPORT.MESSAGE.TYPE.TEXT,
+                style: 'strong',
+                text: ownerEmail === currentUserEmail ? 'You' : ownerEmail,
+            },
+            {
+                type: CONST.REPORT.MESSAGE.TYPE.TEXT,
+                style: 'normal',
+                text: ' created this report',
+            },
+        ],
+        person: [
+            {
+                type: CONST.REPORT.MESSAGE.TYPE.TEXT,
+                style: 'strong',
+                text: lodashGet(allPersonalDetails, [currentUserEmail, 'displayName'], currentUserEmail),
+            },
+        ],
+        automatic: false,
+        sequenceNumber: 0,
```

Comment:
Can we get rid of this now? Won't all new clients that get this update already be keying by reportActionID

Reply:
Yeah, we could probably get rid of it now. My plan was to mostly leave the old sequence numbers in place for now and handle them in a follow-up PR, but I guess that plan reduces to just leaving a bunch of unused code in the repo, which doesn't make a ton of sense either.

Reply:
Done, and boy was that fun. The only references you will find to reportAction sequenceNumbers or clientIDs anywhere in the code is now in the migration lib.